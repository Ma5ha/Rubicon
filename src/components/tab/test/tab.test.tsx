import React from "react";
import renderer from "react-test-renderer";
import NavigationWarper from "../tab";

it("renders elements <a/> <p/> <h1/>", () => {
  const tree = renderer
    .create(
      <NavigationWarper>
        <h1>Rubicon</h1> <a>Rubicon</a> <p>Rubicon</p>
      </NavigationWarper>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
