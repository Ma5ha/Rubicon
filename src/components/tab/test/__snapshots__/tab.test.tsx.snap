// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders elements <a/> <p/> <h1/> 1`] = `
<div
  className="navigation"
>
  <h1>
    Rubicon
  </h1>
   
  <a>
    Rubicon
  </a>
   
  <p>
    Rubicon
  </p>
</div>
`;
